test/unit_tests/augmentation/conftest.py

```diff
@@ -26,3 +33,24 @@ def random_batch(rng_seed, batch_size=5):
     rng = check_random_state(rng_seed)
     X = torch.from_numpy(rng.random((batch_size, 66, 51))).float().to(device)
     return X, torch.zeros(batch_size)
+
+
+class MockModule(torch.nn.Module):
+    def __init__(self, preds):
+        super().__init__()
+        self.preds = to_tensor(preds, device='cpu')
+        self.linear = torch.nn.Linear(5, 5)
+
+    def forward(self, x):
+        return self.preds
+
+
+@pytest.fixture
+def mock_clf():
```

Comment:
I would rename it into augmented_mock_clf as you have added a transform to it

Reply:
Done